File: 00-algorithm/solve-java/src/main/java/dev/chan99k/BOJ_1001.java
package dev.chan99k;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class BOJ_1001 {

	public static void main(String[] args) throws IOException {
		FastReader fr = new FastReader();
		int a = fr.nextInt();
		int b = fr.nextInt();
		System.out.println(a - b);
	}

	static class FastReader {
		private final BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		private String[] tokens;
		private int index;

		String next() throws IOException {
			while (tokens == null || index >= tokens.length) {
				tokens = br.readLine().split(" ");
				index = 0;
			}
			return tokens[index++];
		}

		int nextInt() throws IOException {
			return Integer.parseInt(next());
		}
	}

}

File: 00-algorithm/solve-kotlin/src/main/kotlin/dev/chan99k/algorithm/BOJ_1001.kt
package dev.chan99k.algorithm

import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.StringTokenizer

object BOJ1001 {
    private val br = BufferedReader(InputStreamReader(System.`in`))
    private var st: StringTokenizer? = null

    private fun next(): String {
        while (st?.hasMoreTokens() != true) {
            st = StringTokenizer(br.readLine())
        }
        return st!!.nextToken()
    }

    private fun nextInt() = next().toInt()

    @JvmStatic
    fun main(args: Array<String>) {
        val a = nextInt()
        val b = nextInt()
        print(a - b)
    }
}
